src/jshint.js

```diff
@@ -213,6 +213,8 @@ var JSHINT = (function() {
     if (state.option.module) {
       if (state.option.strict === true) {
         state.option.strict = "global";
+      } else if (!state.option.strict) {
+        state.option.strict = "global?";
```

Comment:
should we document `global?` or was the intention that it be internal only?
What does it mean - I'm wondering if it would be better having a different word e.g. `ignore` or `global-implied` or something else?